lightningd/plugin.h

```diff
@@ -88,6 +88,9 @@ struct plugin {
 	/* Notification topics that this plugin has registered with us
 	 * and that other plugins may subscribe to. */
 	const char **notification_topics;
+
+	/* Plugin addendum (e.g. configs) it wants shown by `listconfigs` */
+	const char *addendum;
```

Comment:
Going to be quite horrible for schema-based RPC bindings, but I like the idea.

Reply:
I would resist this strongly until proven necessary.

It's not consistent to have things which are not configs in listconfigs.  If it's in listconfigs (and, yes, adding plugin randomness to listconfigs was my mistake), it should be valid as cmdline/config file.

Reply:
> I would resist this strongly until proven necessary.

Agreed,  will drop it.
Maybe something for another PR and then add it to `plugin list` instead of `listconfigs`.